tests/browser/stringify.browser.js

```diff
@@ -30,31 +30,7 @@ test('fake stringify', function (t) {
 
   var a = {}
   a.a = a
-  t.equal(stringify(a), undefined, 'Stringifying a circular object returns empty string')
-
-  Array.prototype.toJSON = function () { return 'bad!' }
-  String.prototype.toJSON = function () { return 'bad!' }
-  Number.prototype.toJSON = function () { return 'bad!' }
-  Date.prototype.toJSON = function () { return 'bad!' }
-  Object.prototype.toJSON = function () { return 'bad!' }
-  Object.prototype.toString = function () { return 'bad!' }
-
-  t.equal(stringify([1, 2, 3]), '[1,2,3]', 'array detection w/o toString')
-
-  t.equal(stringify(arr), '[0,1,"asdf",null,null,"weee"]', 'Array with bad toJSON')
-  t.equal(stringify(undefined), undefined, 'undefined with bad toJSON')
-  t.equal(stringify(null), 'null', 'null with bad toJSON')
-  t.equal(stringify(123), '123', 'number with bad toJSON')
-  t.equal(stringify(obj), '{"a":123,"c":"b","f":"asdf","n":null}', 'obj with bad toJSON')
-  t.equal(stringify(obj2), '{"other":222}', 'obj w/ prototype with bad toJSON')
-  t.equal(stringify(F), undefined, 'function with bad toJSON')
-  t.ok(stringify(obj3) === '{"stringified":"{\\"a\\":123,\\"c\\":\\"b\\",\\"f\\":\\"asdf\\",\\"n\\":null}"}', 'stringified object with bad toJSON')
-
-  delete Array.prototype.toJSON
-  delete String.prototype.toJSON
-  delete Number.prototype.toJSON
-  delete Date.prototype.toJSON
-  delete Object.prototype.toJSON
+  t.equal(stringify(a), '{}', 'Stringifying a circular object returns empty object')
```

Comment:
Have we verified with ingest or whoever parses the JSON how they want us to stringify these circ refs?

Reply:
not even sure if we were able to report it before.  The previous test indicated that it would just be "undefined", not even a string

Reply:
Without this change, the data is not a valid json string. I think we are good to move forward with this.